Casks/teamviewer.rb

```diff
@@ -3,11 +3,15 @@
   sha256 :no_check
 
   url "https://download.teamviewer.com/download/TeamViewer.dmg"
-  appcast "https://download.teamviewer.com/download/update/macupdates.xml?id=0&lang=en&version=14.7.1965&os=macos&osversion=10.15.1&type=1&channel=1"
   name "TeamViewer"
   desc "Remote access and connectivity software focused on security"
   homepage "https://www.teamviewer.com/"
 
+  livecheck do
+    url "https://download.teamviewer.com/download/update/macupdates.xml?id=0&lang=en&version=15.9.4&os=macos&osversion=10.15.1&type=1&channel=1"
```

Comment:
I think we should also make the macOS version dynamic. 
```suggestion
    url "https://download.teamviewer.com/download/update/macupdates.xml?id=0&lang=en&version=#{version}&os=macos&osversion=10.15.1&type=1&channel=1"
```

Reply:
@SMillerDev Yes good call. Thanks.